docs/sources/datahub-cloud-event-source.md

```diff
@@ -0,0 +1,114 @@
+# DataHub Cloud Event Source
+
+## Prerequisites
+
+### Compatibility
+
+The **DataHub Cloud Event Source** is only compatible with versions of DataHub Cloud above  `v0.3.7`.
+
+### Privileges
+
+By default, users do not have access to the Events API of DataHub Cloud. In order to access the API, the user or service account
+associated with the access token used to configure this events source _must_ have the `Get Platform Events` platform privilege, which
```

Comment:
why not grant this to all Admins? Admins already have the ability to read all metadata. This seems like it will be a constant source of frustration.

Reply:
Currently, the root user does have it but they are the only ones. We can grant to admins once we validate that everything is working. I agree it could be frustrating

Reply:
e.g. the Admin user